docs/trouble_shooting.md

```diff
@@ -0,0 +1,20 @@
+# Trouble shooting
+
+## get log of clusterapi-controllers containers
```

Comment:
s/get log of clusterapi-controllers containers/Get log of clusterapi-controllers containers